pkgs/by-name/es/eslint/package.nix

```diff
@@ -30,11 +30,16 @@ buildNpmPackage' rec {
     cp ${./package-lock.json} package-lock.json
   '';
 
-  npmDepsHash = "sha256-Nrcld0ONfjdSh/ItdbDMp6dXVFKoj83aaoGXDgoNE60=";
+  npmDepsHash = "sha256-F3EUANBvniczR7QxNfo1LlksYPxXt16uqJDFzN6u64Y=";
 
   dontNpmBuild = true;
   dontNpmPrune = true;
 
+  postFixup = ''
+    # Remove broken symlink
+    rm $out/lib/node_modules/eslint/node_modules/eslint-config-eslint
```

Comment:
I think we want it to point to the local packages folder in src 
https://github.com/eslint/eslint/tree/main/packages/eslint-config-eslint